packages/profiling-node/src/integration.ts

```diff
@@ -37,129 +332,29 @@ export const _nodeProfilingIntegration = (() => {
 
   return {
     name: 'ProfilingIntegration',
+    profiler: new ContinuousProfiler(),
```

Comment:
m: Why is this public API? If we need it for tests, we can make it private API and just access it via `integration['_profiler']`, TS will be fine with that.

also ideally we only call the constructor in `case 'continuous'` below.

Reply:
Yeah, I dont know exactly how this will work yet as we are standardizing the docs, but we'll want to expose something like Sentry.profiler.start and Sentry.profiler.stop which would result in noop calls if used in the wrong mode. Right now this is just done through the implicit binding of the sentry client, but we'll want something clearer and easier to follow here